packages/components/segmented-control/tests/segmented-control.test.tsx

```diff
@@ -99,7 +99,7 @@ describe("<SegmentedControl />", () => {
 
   test("SegmentedControl with non-SegmentedControlButton children renders correctly", async () => {
     const { container } = render(
-      <SegmentedControl data-testid="SegmentedControl" isDisabled>
+      <SegmentedControl data-testid="SegmentedControl" disabled>
```

Comment:
@nathanlao 
We should ensure that the props with `@depricated` are also working, so please do not change the props in the test code.